src/test/java/com/epam/tc/hw7/ex1/TestSubmitMetalsColorsData.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.tc.hw7.ex1;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.tc.hw7.AbstractBaseTest;
+import com.epam.tc.hw7.ex1.pages.MetalColorPage;
+import org.testng.annotations.Test;
+
+public class TestSubmitMetalsColorsData extends AbstractBaseTest {
+    MetalColorPage metalColorPage = new MetalColorPage();
```

Comment:
This is not the best way to initialise JDI page, check PageFactory::initSite method please